src/components/Todo/TodoFooter/FooterLink/FooterLink.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+type Props = {
+  dataCy: string,
+  href: string,
+  className: string,
+  text: string,
+  onClick: () => void,
+};
+
+const FooterLink: React.FC<Props> = ({
+  dataCy, href, className, onClick, text,
+}) => {
```

Comment:
```suggestion
const FooterLink: React.FC<Props> = ({
  dataCy, 
  href, 
  className, 
  onClick, 
  text,
}) => {
```
Please fix all cases